docs/spfx/build-a-hello-world-web-part.md

```diff
@@ -1,171 +1,160 @@
+   
+## Preview the web part
+To preview your web part you need to build and run it in a local web server. 
 
-Switch to your console, make sure you are still in the `helloworld-webpart` directory and type the following to build and preview your web part:
+* Switch to your console, make sure you are still in the **helloworld-webpart** directory and enter the following command to build and preview your web part:
 
 ```
 gulp serve
 ```
 
->If you see an error `cannot find module es6-promise`, please execute the command in the project folder to install the es-promise npm module: `npm i es6-promise` and then do `gulp serve` again.
-
-
-> Visual Studio Code provides built-in support for gulp and other task runners. You can press `Ctrl+Shift+B` if you are in Windows or `Cmd+Shift+B` if you are in Mac within Visual Studio Code to debug and preview your web part. 
-
 This command will execute a series of gulp tasks to create a local, Node-based HTTP server on 'localhost:4321' and launch your default browser to preview web parts from your local dev environment.
 
 ![Gulp serve web part project](./images/helloworld-wp-gulp-serve.png)
 
-### SharePoint Workbench
-SharePoint Workbench is a developer design surface to quickly preview and test web parts without needing to deploy them in SharePoint. SharePoint Workbench includes the new client-side page and the new client-side canvas in which you can add, delete and test your web parts in development.
-
-![SharePoint Workbench running locally](./images/sp-workbench.png)
-
-To add the `HelloWorld` web part, click on the add (`plus icon`) button. Add button opens the new `Toolbox` where you can see a list of web parts available for you to add. The list will include the `HelloWorld` web part as well other web parts available locally in your developer environment.
-
-![SharePoint Workbench toolbox in localhost](./images/sp-workbench-toolbox.png)
+>**Note:** If you get the error **cannot find module es6-promise**, run the following command in the project folder to install the es-promise npm module: `npm i es6-promise` and then run `gulp serve` again.
 
-Click on the `HelloWorld` to add the webpart to the page:
-
-![HelloWorld web part in SharePoint Workbench](./images/sp-workbench-helloworld-wp.png)
+SharePoint client-side development tools use [gulp](http://gulpjs.com/) as the task runner to handle build process tasks such as:
+* Bundle and minify JavaScript and CSS files.
+* Run tools to call the bundling and minification tasks before each build.
+* Compile SASS files to CSS.
+* Compile TypeScript files to JavaScript.
 
-Congratulations! You have just added your first client-side web part to a client-side page.
+If you are new to gulp, you can read [Using Gulp](http://docs.asp.net/en/latest/client-side/using-gulp.html) which describes using gulp with Visual Studio in conjunction with building ASP.NET 5 projects.
```

Comment:
Revise as "For more information about using gulp with Visual Studio, see Using Gulp."